src/app/shared/detective-item/detective-item/detective-item.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="detective">
+  <div class="detective__medal {{ index === 0 ? 'font-supernova' : index === 1 ? 'font-purple' : '' }}">
```

Comment:
I find `[class.font-supernova]="index===0" [class.font-purple]="index===1"` more practicable for conditional classes instead of chaining ternary expressions.